src/encoded/upgrade/library.py

```diff
@@ -103,8 +103,6 @@ def library_4_5(value, system):
 @upgrade_step('library', '5', '6')
 def library_5_6(value, system):
     # http://redmine.encodedcc.org/issues/2491
-    if not value['depleted_in_term_name']:
```

Comment:
Why remove this?

Reply:
I am not sure why this line was there in the first place, the tests for upgrade (8->9) fail if this line is in. Reading the upgrade (5>6) the line I removed doesn't make any sense to me.